owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/executor/SimpleHibernateBatch.java

```diff
@@ -0,0 +1,128 @@
+package fr.openwide.core.jpa.batch.executor;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.support.TransactionCallback;
+
+import com.google.common.collect.Lists;
+import com.querydsl.core.types.dsl.PathBuilder;
+import com.querydsl.jpa.impl.JPAQuery;
+
+import fr.openwide.core.commons.util.functional.Joiners;
+import fr.openwide.core.jpa.business.generic.model.GenericEntity;
+import fr.openwide.core.jpa.business.generic.model.QGenericEntity;
+import fr.openwide.core.jpa.exception.ServiceException;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class SimpleHibernateBatch extends AbstractBatch<SimpleHibernateBatch> {
```

Comment:
I mean that this class does exactly the same as MultithreadedBatch, only with two more features that are switched off by default. So by default, this class does exactly the same as MultithreadedBatch. Thus we could just have one class, and let the user choose if he wants to enable the indexing features.